packages/styling/src/styles/DefaultFontStyles.ts

```diff
@@ -84,7 +84,7 @@ function _getFontBaseUrl(): string {
   let win = getWindow();
 
   // tslint:disable-next-line:no-string-literal no-any
-  let fabricConfig: IFabricConfig = win ? win['FabricConfig'] : undefined;
+  let fabricConfig: IFabricConfig = win ? (win as any).FabricConfig : undefined;
```

Comment:
Also not super important but this can use `?.` now
```suggestion
  let fabricConfig: IFabricConfig = win?.FabricConfig;
```